src/modules/sms/service.js

```diff
@@ -0,0 +1,88 @@
+const { INVALID_SMS_CODE_MESSAGE, INVALID_PHONE_NUMBER } = require('./constants');
+const db = require('../../models');
+const Hashids = require('hashids');
+const { SMS, AUTH } = require('../../config');
+const { ValidationError } = require('../../utils/errors');
+const userService = require('../user/service');
+const authService = require('../auth/service');
+const { sendSMS } = require('../../helpers/twilio');
+const { Op } = require('sequelize');
+const { DateTime } = require('luxon');
+
+const hashids = new Hashids(SMS.SALT);
+
+const verifySMSCode = async (userCode) => {
+  const [id] = hashids.decode(userCode);
+  if (!id) {
+    throw new ValidationError(INVALID_SMS_CODE_MESSAGE);
+  }
+
+  const code = await getSMSCode({
```

Comment:
Better to create extra method `getActiveSMSCode` and wrap all the conditions into that